lib/rails/generators/alchemy/install/files/application.html.erb

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%= @page.language_code %>">
```

Comment:
One thing we should do some recherche on is: Do we need the country code part? Or is the language code sufficient? Maybe it is. Then `@page.language_code` is actually the correct one 😅